src/ruby/spec/generic/client_stub_spec.rb

```diff
@@ -50,7 +50,7 @@ def check_op_view_of_finished_client_call(op_view,
 
   expect { op_view.start_call }.to raise_error(RuntimeError)
 
-  sanity_check_values_of_accessors(op_view,
+  consistency_check_values_of_accessors(op_view,
```

Comment:
There are just two style check errors here: the style checker wants these multi-line parameters to be aligned with the top one.